.gitignore

```
@@ -11,3 +11,4 @@ msbuild.log
 *.tmp
 ~*.docx
 ~*.xlsx
+packages/*
```

This doesn't seem to be configured correctly, since the PR includes all the assemblies in your nuget cache. 
```suggestion
packages
```

---

Thanks for your suggestion, all the current assemblies were added when .gitignore was not configured before, we will remove all the assemblies in the next update. and we'll resolve heaths's comments ASAP.